2021/01/22 17:35:40 : Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 665, in storage_data
    getattr(self.get_source_cart(), 'finish_storage_data')()
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/basecart.py", line 4043, in finish_storage_data
    folder_upload = self.get_folder_upload(self._migration_id)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/basecart.py", line 4055, in get_folder_upload
    folder_upload = root_path + '/' + DIR_UPLOAD + '/' + get_config_ini('src', 'file', file='test.ini')
TypeError: can only concatenate str (not "NoneType") to str

2021/01/22 17:35:51 : products_errors 9: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:51 : products_errors 10: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:51 : products_errors 16: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:51 : products_errors 17: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 18: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 19: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 20: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 22: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 23: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 24: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 25: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 37: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 39: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 41: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 42: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 43: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 44: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 47: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 48: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

2021/01/22 17:35:52 : products_errors 49: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/custom.py", line 445, in convert_product_export
    pl['description'] = product_data['products_description']
KeyError: 'products_description'

